cache/edge_lambdas/src/redirects.ts

```diff
@@ -23,20 +23,22 @@ export const literalRedirects: Record<string, string> = {
   '/access': '/pages/accessibility',
   '/alice-anderson':
     '/exhibitions/alice-anderson--memory-movement-memory-objects',
-  '/articles/aids-posters-0': '/articles/aids-posters',
-  '/articles/hysteria': '/articles/what-is-hysteria', // "hysteria" already taken by ZN-ELxEAACMABO5a in former UID.
+  '/articles': '/search/stories',
```

Comment:
Will this cause a conflict with line 188?

Reply:
I'm not clear on where they get stored/how it works, I'd love to understand that better 🙈

Reply:
As discussed just now, conflict won't be an issue.

Reply:
No, line 188 will only ever be invoked for things that aren't in this list